src/notifications/settings.ts

```diff
@@ -507,9 +506,15 @@ const unsubscribeWalletFromSingleNotificationTopic = async (
   topic: ValueOf<typeof NotificationTopic>
 ) => {
   Logger.log(
-    `Notifications: unsubscribing ${type}:${address} from [ ${topic} ]`
+    `Notifications: unsubscribing ${type}:${address} from [ ${topic.toUpperCase()} ]`
   );
   messaging().unsubscribeFromTopic(
     `${type}_${chainId}_${address.toLowerCase()}_${topic}`
   );
+  analytics.track('Changed Notification Settings', {
```

Comment:
nit but maybe we want to move this into `notifications/analytics.ts` from juba's pr?

Reply:
yep i have a note about that. will wait for his PR to be merged and then i'll rebase and move it